frontend/components/home.jsx

```diff
@@ -57,56 +79,260 @@ class Home extends React.Component {
     this.createInterval();
   }
 
+  playNextSongDifferent() {
```

Comment:
Change this name from what I remember this isn't actually playing the next song so the name is not appropriate.